src/include/dhcp-server/dialogs.rb

```diff
@@ -47,44 +53,66 @@ def ReadDialog
     end
 
     # Write settings dialog
-    # @return `abort if aborted and `next otherwise
+    #
+    # @return [Symbol] :next whether configuration is saved successfully
+    #                  :back if user decided go back to change settings
+    #                  :abort otherwise
     def WriteDialog
-      Builtins.y2milestone("Running write dialog")
-      Wizard.RestoreHelp(Ops.get(@HELPS, "write", ""))
-      ret = DhcpServer.Write
-      if ret && restart_after_writing?
-        # Restart only if it's already running
-        DhcpServerUI.service.try_restart
-      end
-      # yes-no popup
-      if !ret &&
-          Popup.YesNo(
-            _("Saving the configuration failed. Change the settings?")
-          )
-        return :back
-      end
-      ret ? :next : :abort
+      help_text = @HELPS.fetch("write") { "" }
```

Comment:
What about `@HELPS.fetch("write", "")` ?

Reply:
Yep, it is more simple and readable. 

I usually use the block notation because [performance: in this case the empty string only will be created if Hash has no the `write` key](https://github.com/JuanitoFatas/fast-ruby/issues/39#issuecomment-103989335). However, I must admit that in this case the difference it will be not significant. So, I will to change it :)